Small tweaks to the Unfettered Jump Drive Trading missions + move the transport hai child mission (#9757)
endless-sky/endless-sky: data/hai/unfettered 0 prologue.txt

				`	"What do you plan to do to the species that inhabits those worlds now?"`
			
			label wanderers

> 			`	"Those worlds are now held by a species of scavengers, who feast on the ruins of proud civilizations. Our scouts tell us that these carrion-feeders have wiped away nearly every Hai artifact, melting down our cities to make metal for their ships and factories, and hiding the scars of our wars beneath newly planted forests. They are an old and strong species, but few in number, and those worlds are ours by right."`

Reviewer: Is this a typo fix? I don't think the previous version is actually incorrect.

Reviewer 2: Ruin and ruins doesn't mean the same thing: ruin's a verb while ruins is a noun.

Author: ```suggestion
			`	"Those worlds are now held by a species of scavengers, who feast on the ruin of proud civilizations. Our scouts tell us that these carrion-feeders have wiped away nearly every Hai artifact, melting down our cities to make metal for their ships and factories, and hiding the scars of our wars beneath newly planted forests. They are an old and strong species, but few in number, and those worlds are ours by right."`
```

		not "Wanderers: Jump Drive Source: done"
		# There is no minimum amount of JD to sell (other than the first 3), except if you failed to prove yourself the first time.
		# "Unfettered Jump Drive 4: failed" represents the amount of JD sold.
		"unfettered: jd to sell" <= "Unfettered Jump Drive 4: failed"
		random < 20 + 8 * "Unfettered Jump Drive 4: failed"
	on offer
		conversation

> 			`As you exit your ship to head to the spaceport you see that an Unfettered Hai<hood> is waiting for you at your hatch.`

Reviewer 3: I think the second <hood> substitution needs to end with a comma, but I can't comment directly on that line here.

@tibetiroka

Reviewer 4: Yeah, a comma would make it much better. Also, this seems to be the only place where the substitution is used, so it should be fine to just edit it.

> 			`As you exit your ship to head to the spaceport you see that an Unfettered Hai<hood> - is waiting for you at your hatch.`

Reviewer 4: ```suggestion
			`As you exit your ship to head to the spaceport you see that an Unfettered Hai<hood> is waiting for you at your hatch.`
```
Honestly, I'm not sure the `<hood>` makes sense even with this fix.

				to display
					"unfettered: blade rodeo" > 1
				goto blade

> 			`	You exit your ship and immediately the engineer starts leading you to the outfitter. "We have weapons to show you for your assistance in providing us with jump drives. For now you will only be allowed to trade for the one, but if you continue to provide assistance to us, you may prove yourself worthy to access the others." They never stop to ask if you are actually interested in new weapons.`

Reviewer 5: ```suggestion
			`	You exit your ship and immediately the engineer leads you to the outfitter. "We have weapons to show you for your assistance in providing us with jump drives. For now you will only be allowed to trade for the one, but if you continue to provide assistance to us, you may prove yourself worthy to access the others." They never stop to ask if you are actually interested in new weapons.`
```

			label meet
			action
				log "Minor People" "Desneeparu" `An Unfettered warrior that knows how to handle a blade. Was tested by her before being given access to new Unfettered weapons. Would have been a goner had she not been holding back.`
			`	She does some kind of salute with her fist before leaving. "I must go now. I hope you will find the weapon you need."`

> 			`	<The engineer> proceeds to take you underground to rows and rows of prototype weapons, some of which are being worked on by engineers. They explain, "We prefer to do most of our testing on the field of battle. Here we only make the most basics of tests before Unfettered captains can get their hands on it. They often start by going to Firelode to put the weapon to its limit, to ensure it will not explode in the middle of action. The more interest in a prototype, the more we will develop it." You keep walking until you enter the outfitter section of the facility, where you notice the items on display are much more advanced than anything you have seen available on the surface: so much so it could almost pass for a 'normal' Hai outfitter, were it not for the overcharged, oppressive feel of the testing grounds.`

Reviewer 4: ```suggestion
			`	<The engineer> proceeds to take you underground to rows and rows of prototype weapons, some of which are being worked on by engineers. They explain, "We prefer to do most of our testing on the field of battle. Here we only make the most basics of tests before Unfettered captains can get their hands on them. They often start by going to Firelode to put the weapon to its limit, to ensure it will not explode in the middle of action. The more interest in a prototype, the more we will develop it." You keep walking until you enter the outfitter section of the facility, where you notice the items on display are much more advanced than anything you have seen available on the surface: so much so it could almost pass for a 'normal' Hai outfitter, were it not for the overcharged, oppressive feel of the testing grounds.`
```

> 			`	<The engineer> proceeds to take you underground to rows and rows of prototype weapons, some of which are being worked on by engineers. They explain, "We prefer to do most of our testing on the field of battle. Here we only make the most basics of tests before Unfettered captains can get their hands on them. They often start by going to Firelode to put the weapon to its limit, to ensure it will not explode in the middle of action. The more interest in a prototype, the more we will develop it." You keep walking until you enter the outfitter section of the facility, where you notice the items on display are much more advanced than anything you have seen available on the surface: so much so it could almost pass for a 'normal' Hai outfitter, were it not for the overcharged, oppressive feel of the testing grounds.`

Reviewer 5: ```suggestion
			`	<The engineer> proceeds to take you underground to rows and rows of prototype weapons, some of which are being worked on by engineers. They explain, "We prefer to do most of our testing on the field of battle. We only make the most basic safety tests here before Unfettered captains can get their hands on them. They often start by going to Firelode to put the weapon to its limit, to ensure it will not explode in the middle of action. The more interest in a prototype, the more we develop it." You keep walking until you enter the outfitter section of the facility, where you notice the items on display are much more advanced than anything you have seen available on the surface: so much so it could almost pass for a 'normal' Hai outfitter, were it not for the overcharged, oppressive feel of the testing grounds.`
```

			action
				set "choice: blaster"
			scene "outfit/hai ionic blaster"

> 			`	"Ah, the Blaster. It is favored by many Shield Beetle captains, because it can easily help them turn the tide of battle without needing to dedicate too much weapon space to it. Especially considering its other effects, its damage potential is serviceable. While it is true that it does not do enough ion damage to properly disable a ship on its own, it is able to jam their weapons and will give you the upper hand in a long fight. This is a more recent innovation, however, and some say it lacks the... finesse of others."`

Reviewer 4: ```suggestion
			`	"Ah, the Blaster. It is favored by many Shield Beetle captains, because it can easily help them turn the tide of battle without needing to dedicate too much weapon space to it. Especially considering its other effects, its damage potential is serviceable. While it is true that it does not do enough ion damage to properly disable a ship on its own, it is able to jam their weapons and will give you the upper hand in a long fight. This is a more recent innovation, however, and some say it lacks the... finesse of others.`
```

				goto backtochoose
			label turret
			action
				set "choice: turret"
			scene "outfit/hai ionic turret"

> 			`	"That is my preferred weapon out of all of them, by far. Unlike the Ion Cannon that is a hybrid weapon meant to damage the enemy from long range whilst applying a good deal of ionization, this one's role is mostly to disable enemy ships. Even a ship with many batteries will cower before this weapon! And due to its high range and fast turn-rate, small ships and even fighters will never bother you again!"`

Reviewer 4: ```suggestion
			`	"That is my preferred weapon out of all of them, by far. Unlike the Ion Cannon that is a hybrid weapon meant to damage the enemy from long range whilst applying a good deal of ionization, this one's role is mostly to disable enemy ships. Even a ship with many batteries will cower before this weapon! And due to its high range and fast turn-rate, small ships and even fighters will never bother you again!`
```
